docs/content/setup/windows.md

```diff
@@ -9,27 +9,27 @@ tags: ["setup", "windows"]
 
 ## Deployment
 
-Run the installer, and you should be met with the following screen:
+After running the installer, the following prompt should appear on the screen:
 
 ![Windows installer screenshot](../../images/setup/windows/installer-screenshot-1.png "Windows installer screenshot")
 
-1. Follow the steps of the installation.
+1. Follow the steps to complete the installation.
 1. Run the Monkey Island by clicking on the desktop shortcut.
 
 ## Troubleshooting
 
-### Missing windows update
+### Missing Windows update
 
-The installer requires [Windows update #2999226](https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows) to be installed. If you’re having trouble running the installer, please make sure to install that update via Windows Update or manually from the link.
+The installer requires [Windows update #2999226](https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows). If you’re having trouble running the installer, please make sure to install the update via Windows Update or manually from the link above.
 
 ### Supported browsers
 
-The Monkey Island supports Chrome (and Chrome-based) browsers. Some Windows Servers only have Internet Explorer installed. Make sure to use Chrome or a similar modern browser. [You can download Google Chrome from here](https://www.google.com/chrome/).
+The Monkey Island supports Chrome (and Chrome-based) browsers. Some Windows Servers only have Internet Explorer installed, so make sure you're using Chrome or a similar modern browser. [You can download Google Chrome here](https://www.google.com/chrome/).
```

Comment:
I think what we're trying to convey here is that "If your Windows server only has Internet Explorer installed, you'll need to install Chrome or a similar modern browser."